mesheryctl/internal/cli/root/system/system.go

```diff
@@ -27,7 +27,11 @@ import (
 )
 
 const (
-	fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	// TODO: `context` support for `system start` change the docker URL to dynamic URL
+	fileURL         = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	manifestsURL    = "https://api.github.com/repos/layer5io/meshery/git/trees/3ba314a0870d5291be6216b4d60d2bc9675a39b2"
```

Comment:
The `manifestsURL` added uses the latest SHA of k8 folder, but this will change every time there's a commit modifying `meshery/install/deployment_yamls/k8s/` folder thus arising the need to modify the SHA in the URL manually every time. Can we do something better?

Reply:
Another way of dynamically setting `manifestsURL` is using GitHub API
fetch https://api.github.com/repos/layer5io/meshery/git/trees/{branch/tag}?recursive=1
find object with `"path": "install/deployment_yamls/k8s"` in the `"tree"` list
use it's `"url"`  as `manifestsURL`

Reply:
To @aneeshnema’s point, is the identification of files on GitHub by release tag a centralized function, yet?

Reply:
Some of the commands are yet to be "context aware". So we need to add that. The current function that fetches the `docker-compose.yaml` file is local to the `reset` command. This needs to be extracted and be used in commands like `system start`. I had a chat with @aneeshnema about this and he will be adding this.

Coming to fetching the Kubernetes manifests, we use GitHub API for this and we need to come up with a better way to fetch files according to the version and channel. We may need to add the manifest files as assets during the release for this.

Reply:
Sounds good on both of your points.